Formula/cassandra-reaper.rb

```diff
@@ -0,0 +1,29 @@
+class CassandraReaper < Formula
+  desc "Management interface for Cassandra"
+  homepage "http://cassandra-reaper.io"
+  url "https://github.com/thelastpickle/cassandra-reaper/releases/download/1.2.2/cassandra-reaper-1.2.2-release.tar.gz"
+  sha256 "720aff69e3205301bc07399afc46dae3568d8effffa3712f1852a169ce9801db"
+
+  def install
+    prefix.install "bin/cassandra-reaper"
+    mv "server/target", "cassandra-reaper"
+    share.install "cassandra-reaper"
```

Comment:
Combine this line & the line above into:

```ruby
pkgshare.install "server/target" => "cassandra-reaper"
```

Reply:
With such a description, the path of the jar file will change and it will not work.